keiko/keiko/keiko.py

```diff
@@ -123,8 +123,8 @@ def generate_report(
         env = {**os.environ, "TEXMFVAR": tmp_dirname}
         output = subprocess.run(cmd, cwd=tmp_dirname, env=env, capture_output=True, check=False)
         logger.info("pdflatex run. [report_id=%s] [template=%s] [command=%s]", report_id, template_name, " ".join(cmd))
-        logger.debug(output.stdout.decode("utf-8"))
-        logger.debug(output.stderr.decode("utf-8"))
+        logger.debug(output.stdout.decode("cp1252").encode("utf-8"))
```

Comment:
Interesting. Is the cp1252 output not related to the used local or something?

Reply:
Not actually sure what happened, I just tried to reproduce with a new clone of the main branch on a Debian 11 VM with locale `en_US.UTF-8` and I can't seem to get this error like I had yesterday. These changes to the debugging logs can be disregarded then I think.

Reply:
I reverted the changes since I can't reproduce.